package.json

```diff
@@ -64,8 +64,11 @@
     "ajv": "^6.1.1",
```

Comment:
@honeydripper If i'm not mistaken, time-stamp module is not needed anymore, is it? If not, please remove from package.json and update yarn.lock. That's all from me.